### PR TITLE
session-02 from daniel_dominguez

### PR DESCRIPTION
**Pull request by dcd000**
Hello.
Here are my session-02 exercises files.

### CODE REVIEW

File: daniel_dominguez/session02/exercise03/lemp/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+dependencies:
+  - name: postgresql
+    version: 0.11.0
+    repository: https://kubernetes-charts.storage.googleapis.com
+    condition: postgresql.enabled
+    import-values:
```
[the remark this answers is not in the document]
Comment (dcd000) [reply daniel_dominguez/session02/exercise03/lemp/requirements.yaml]: fixed in 0bc20cfb720f9ac64b4bfb5f1664f74f27edc374

File: daniel_dominguez/session02/exercise02/lemp/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      initContainers:
+      - name: gitclone
+        image: alpine/git
+        command: ["sh", "-c", "git clone https://github.com/bitnami-labs/k8s-training-resources/ /app; cd /app; git pull;"]
+        volumeMounts:
+        - name: nginx-data
+          mountPath: /app
+      containers:
+      - name: nginx
+        image: "bitnami/nginx:latest"
```
[the remark this answers is not in the document]
Comment (dcd000) [reply daniel_dominguez/session02/exercise02/lemp/templates/deployment.yaml]: fixed in 0bc20cfb720f9ac64b4bfb5f1664f74f27edc374

File: daniel_dominguez/session02/exercise01/lemp/values.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
+# Default values for lemp exercise-01.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
```
[the remark this answers is not in the document]
Comment (tompizmor) [reply daniel_dominguez/session02/exercise01/lemp/values.yaml]: I think I was referring to the nginx image but I see that you are getting the image name from the values now

**Comment by tompizmor** [comment]
hey @dcd000, I was going to merge this PR but I saw that you have pushed exercise of the session2, could you create a different PR for that so I can merge this?

**Comment by dcd000** [comment]
Do you mean rename session02 dir as session-02?

**Pull request merged**